src/components/conference/lcc3-schedule.json

```diff
@@ -0,0 +1,200 @@
+[
+  {
+    "est": "7:00",
+    "utc": "13:00",
+    "name": "Johanna Koester",
+    "email": "koester@us.ibm.com",
```

Comment:
Let's exclude the speakers email addresses form the website.  They should not be published

Reply:
Roger that.